ttkthemes/themed_tk.py

```diff
@@ -44,13 +44,21 @@ def __init__(self, *args, **kwargs):
         # Set initial theme
         if theme is not None and theme in self.get_themes():
             self.set_theme(theme, toplevel, background)
+        elif theme is None and background is not None:
+            self.set_background(background)
         self.__init__toplevel = tk.Toplevel.__init__
+        
+    def set_background(self, background):
+        self.config(background=background)
+        ttk.Style(self).configure(".", background=background)
```

Comment:
This is a theme option that I would prefer to set in the theme `Tcl` files instead for themes for which it is appropriate and has not already been set. If the user wants to change the background color of all the widgets in the instance, they should use this method themselves.